internal/featuregates/featuregates.go

```diff
@@ -13,10 +14,12 @@ type FeatureGates struct {
 	ConfigMapName string
 }
 
+const (
+	TimeTravelFeatureGate = "timeTravel"
+)
+
 var DefaultFeatureGates = map[string]bool{
-	"timeTravel":              false,
-	"quantumEntanglementSync": false,
-	"autoHealingWithAI":       true,
```

Comment:
If we are removing here, do you mind removing from `config/samples/featuregates.yaml` too?

Reply:
Also from docs/FEATURE-GATES.md

Reply:
makes sense